use-timescale/migration/pg-dump-and-restore.md

```diff
@@ -8,182 +8,216 @@ tags: [recovery, logical backup, pg_dump, pg_restore]
 
 import ConsiderCloud from "versionContent/_partials/_consider-cloud.mdx";
 
-# Migrations using `pg_dump` and `pg_restore`
+# Migrate using `pg_dump` and `pg_restore`
 
-You can backup and restore an entire database or individual hypertables using
-the native PostgreSQL [`pg_dump`][pg_dump] and [`pg_restore`][pg_restore]
-commands. This works even for compressed hypertables, without having to
-decompress the chunks before you begin. You can also use this method to migrate
-data from one database to another.
+It is possible to migrate from self-hosted PostgreSQL or TimescaleDB to
+Timescale using the native PostgreSQL [`pg_dump`][pg_dump] and
+[`pg_restore`][pg_restore] programs. This works for compressed hypertables,
+without having to decompress data before you begin.
 
-<Highlight type="note">
-Timescale handles backups automatically by default. You do not need to manually
-back up your content. For more information about backups in Timescale, see the
-[backup and restore section](https://docs.timescale.com/use-timescale/latest/backup-restore-cloud/).
+<Highlight type="warning">
+Timescale does not allow having multiple databases per instance.
 </Highlight>
 
-Upgrades between different versions of TimescaleDB can be done in place; you
-don't need to backup and restore your data. See
-the [upgrading instructions][timescaledb-upgrade].
-
 <Highlight type="warning">
-If you are using this `pg_dump` backup method regularly, make sure you keep
-track of which versions of PostgreSQL and TimescaleDB you are running. For more
-information, see the
-[Troubleshooting section](https://docs.timescale.com/self-hosted/latest/self-hosted/backup-and-restore/troubleshooting/).
+A long-running `pg_dump` against a database can cause various issues due to the
+types of locks that `pg_dump` takes. Consult the troubleshooting section
+[Dumping and locks][dumping-and-locks] for more details.
+</Highlight>
+
+## Migrating from TimescaleDB to Timescale
+
+If you are already using TimescaleDB hosted in your own cloud, and would like
+to move your data to the Timescale cloud service, follow these instructions.
+
+<Highlight type="note">
+When dumping and restoring with `pg_dump` and `pg_restore`, the version of the
+TimescaleDB extension must be the same in the source and target databases. If
+this is not the case, subtle data corruption issues may arise.
 </Highlight>
 
-## Back up your entire database
+### Dump the source database
 
-You can perform a backup using the `pg_dump` command at the command prompt. For
-example, to backup a database named `tsdb`:
+Dump the roles from the source database (only necessary if you're using roles
+other than the default `postgres` role in your database):
 
 ```bash
-pg_dump -Fc -f tsdb.bak tsdb
+pg_dumpall -d "$SOURCE" \
```

Comment:
Should we the whole section about sed and what each flag means?

Reply:
I think you're missing a verb here?

Reply:
Should we ADD ... :D